pkg/webhook/validatingwebhook/validate_vm_request_test.go

```diff
@@ -0,0 +1,228 @@
+package validatingwebhook
+
+import (
+	"bytes"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"text/template"
+
+	"github.com/codeready-toolchain/member-operator/pkg/webhook/validatingwebhook/test"
+
+	userv1 "github.com/openshift/api/user/v1"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+func TestHandleValidateVMAdmissionRequestBlocked(t *testing.T) {
+	v := newVMRequestValidator(t, "johnsmith")
+	// given
+	ts := httptest.NewServer(http.HandlerFunc(v.HandleValidate))
+	defer ts.Close()
+
+	// when
+	resp, err := http.Post(ts.URL, "application/json", bytes.NewBuffer(newCreateVMAdmissionRequest(t, VMAdmReviewTmplParams{"CREATE", "johnsmith"}, createVMWithRunStrategyJSONTmpl)))
+
+	// then
+	require.NoError(t, err)
+	body, err := io.ReadAll(resp.Body)
+	defer func() {
+		require.NoError(t, resp.Body.Close())
+	}()
+	require.NoError(t, err)
+	test.VerifyRequestBlocked(t, body, "this is a Dev Sandbox enforced restriction. Configuring RunStrategy is not allowed", "b6ae2ab4-782b-11ee-b962-0242ac120002")
+}
+
+func TestValidateVMAdmissionRequest(t *testing.T) {
```

Comment:
Not something to address now - but should we have all those tests going through the handler ? I mean merging this one with the `TestHandleValidateVMAdmissionRequestBlocked` into something like `TestHandleValidateVMAdmissionRequest`.

Reply:
You're right, I should have done it that way. Done 71f6795892683a2bf13289a9f90fae7d916b58eb